pages/apis/webhooks/agent_events.md.erb

```diff
@@ -8,23 +8,23 @@
 <tbody>
   <tr>
     <th><code>agent.connected</code></th>
-    <td>An agent has connected</td>
+    <td>An agent has connected to the API</td>
   </tr>
   <tr>
     <th><code>agent.lost</code></th>
-    <td>An agent has been marked as lost  </td>
+    <td>An agent has been marked as lost. This happens when Buildkite stops receiving heartbeats from the agent.</td>
```

Comment:
Worth checking if we talk about heartbeats elsewhere. In this case we don't use the term else where. Is this the term we should be using. Would "pings" work?

Reply:
Oh yeah "pings" is probably better
```suggestion
    <td>An agent has been marked as lost. This happens when Buildkite stops receiving pings from the agent.</td>
```